### PR TITLE
User can select hotel

### PR DESCRIPTION
**Pull request by EevanR**
[PT LINK](https://www.pivotaltracker.com/story/show/171209589)

As a user
In order to finalize my trip
I want to be able to select a hotel

- add hotel list component
- render hotel index data
- send id of selected hotel to back end
- send id of selected hotel to back end

Screen Shot 2020-02-11 at 5 08 44 PM

### CODE REVIEW

File: src/components/Activities.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { addActivityType } from "../modules/destination.js";
 
 const Activities = props => {
   const [activityType, setActivityType] = useState(null);
-  const [actTimes, setActTimes] = useState(null);
+  const [activity_visits, setActivity_visits] = useState(null);
```
Comment (mantanquzzmo) [inline src/components/Activities.jsx:8]: activityVisits - but I got your back. Fixed in PR to be merged into this one.

File: src/components/HotelsList.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import { getHotels } from "../modules/destination.js";
+import { Map, InfoWindow, GoogleApiWrapper, Marker } from "google-maps-react";
+import { Button } from "semantic-ui-react";
+import { chooseHotel } from "../modules/destination.js";
+
+const HotelsList = props => {
+  const [gotHotelsData, setGotHotelsData] = useState(false);
+  const [showingInfoWindow, setShowingInfoWindow] = useState(false);
+  const [activeMarker, setActiveMarker] = useState({});
+  const [selectedPlace, setSelectedPlaces] = useState({});
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: is this place or places?

File: src/components/HotelsList.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  const [hotelMessage, setHotelMessage] = useState("");
+
+  const getHotelsShowData = async () => {
+    const hotelsData = await getHotels(
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: suggest to keep it consistent with just response. also less confusion with similar variablenames

File: src/components/Activities.jsx (hunk 1 above)
Comment (EevanR) [reply src/components/Activities.jsx:8]: ok ok tack

File: src/components/HotelsList.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+          </div>
+        </div>
+      );
+    });
+  }
+
+  let markers;
+  
+  if (gotHotelsData) {
+    const onMarkerClick = (props, marker, e) => {
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: e looks unused

File: src/components/HotelsList.jsx (hunk 2 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: good spot, places

File: src/components/HotelsList.jsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+          name={marker.name}
+          position={{ lat: marker.lat, lng: marker.lng }} />
+      )
+    })
+  }
+  
+  return (
+    <>
+      <div className="ui stackable four column grid">
+        {hotelCard ? (
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: use ```{hotelCard && (expression)}``` then you dont need a false path

File: src/components/HotelsList.jsx [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+          <div id="divider">
+            <p>{hotelMessage}</p>
+          </div>
+        ) : null}
+        {hotelCard}
+      </div>
+      <div id="embed-map">
+        {hotelCard ? (
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: same goes for this one

File: src/components/HotelsList.jsx [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+              <InfoWindow
+                marker={activeMarker}
+                visible={showingInfoWindow}>
+                <div>
+                  <h4>{selectedPlace.name}</h4>
+                </div>
+              </InfoWindow>
+            </Map>
+          </div>
+        ) : null}
```
Comment (mantanquzzmo) [inline src/components/HotelsList.jsx]: and this badboy i suspect

File: src/components/HotelsList.jsx (hunk 3 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: changed

File: src/components/HotelsList.jsx (hunk 4 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: correct, gone

File: src/components/HotelsList.jsx (hunk 5 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: yup, changed

File: src/components/HotelsList.jsx (hunk 6 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: changed

File: src/components/HotelsList.jsx (hunk 7 above)
Comment (EevanR) [reply src/components/HotelsList.jsx]: gone

**Review by mantanquzzmo — APPROVED** [review]

**Pull request merged**